packages/block-editor/src/components/block-mover/index.native.js

```diff
@@ -2,29 +2,52 @@
  * External dependencies
  */
 import { first, last, partial, castArray } from 'lodash';
+import React, { useRef, useState } from 'react';
```

Comment:
Don't think we need to import `React` here. And I think we should import useRef and useState from '@wordpress/element' to be consistent with code elsewhere in the project.

Reply:
You're right... although I find it funny that we're literally rewriting the wheel here.

Reply:
It felt weird to me when I first saw us doing this too. I assume the reason we do this is to reduce our vulnerability to changes in react, i.e. reasons [along these lines](https://softwareengineering.stackexchange.com/questions/107338/using-third-party-libraries-always-use-a-wrapper). There's certainly a not-insignificant complexity trade-off we incur by doing that though.